pkg/initialise/initialise.go

```diff
@@ -0,0 +1,128 @@
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/download/yamlcreator"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/util"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/util/files"
+)
+
+//CreateTemplate Creates a blank set of monaco folders and files for each supported API
+func CreateTemplate(workingDir string, createSpecificAPI string) error {
+
+	util.Log.Info("Creating Config for APIs: %v", createSpecificAPI)
+
+	apiList, err := getAPIList(createSpecificAPI)
+	if err != nil {
+		return err
+	}
+
+	workingDir = filepath.Clean(workingDir)
+	util.Log.Info("Initialising Monaco Demo Folders")
+	projectsFolder := filepath.Join(workingDir, "projects")
+	demoProjectFolder := filepath.Join(workingDir, "projects", "baseconfig")
+
+	creator := files.NewDiskFileCreator()
+
+	// Create environments.yaml file
+	environmentsContent := `environment1:
+		    - name: "environment1"
+		    - env-url: "{{ .Env.ENVIRONMENT_ONE }}"
+		    - env-token-name: "{{ .Env.TOKEN_ENVIRONMENT_ONE }}"`
+	_, err = creator.CreateFile([]byte(environmentsContent), workingDir, "environments", ".yaml")
+
+	if err != nil {
+		util.Log.Error("Error creating environments.yaml file")
+	}
+	util.Log.Info("Created File: environments.yaml")
+
+	// Create /projects folder
+	fullpath, err := creator.CreateFolder(projectsFolder)
+	if err != nil {
+		util.Log.Error("Error creating top level projects folder. %v - %v", fullpath, err)
+		return err
+	}
+	util.Log.Info("Created Folder: %v", projectsFolder)
+
+	// Create /projects/baseconfig folder
+	creator.CreateFolder(demoProjectFolder)
+	if err != nil {
+		util.Log.Error("Error creating top level projects folder. %v - %v", fullpath, err)
+		return err
+	}
+	util.Log.Info("Created Folder: %v", demoProjectFolder)
+
+	// For each allowed API, create the relevant folder
+
+	for folderName := range apiList {
+		configTypeFolderPath := filepath.Join(demoProjectFolder, folderName)
+		creator.CreateFolder(configTypeFolderPath)
+		if err != nil {
+			util.Log.Error("Error creating /projects/%v >> %v - %v", folderName, fullpath, err)
+			return err
+		}
+		util.Log.Info("Created Folder: %v", configTypeFolderPath)
+
+		config := yamlcreator.NewYamlConfig()
+		/* Add config like:
+				*  config:
+				*  - folderName: folderName.json
+				*  folderName:
+				*  - name: folderName-one
+				*
+				* For example:
+		        * config:
+		        * - alerting-profile: alerting-profile.json
+		        * alerting-profile:
+		        * - name: alerting-profile-one
+		*/
+		config.AddConfig(folderName, folderName+"-one")
+
+		/* Create file as "folderName.yaml"
+		 * eg. alerting-profile.yaml
+		 */
+		err := config.CreateYamlFile(creator, configTypeFolderPath, folderName)
+		if err != nil {
+			util.Log.Error("Error creating YAML file %v", err)
+		}
+
+		util.Log.Info("  Created File: %v/%v%v", configTypeFolderPath, folderName, ".yaml")
+		creator.CreateFile(nil, configTypeFolderPath, folderName, ".json")
```

Comment:
So, if I understand correctly, the json file is empty?

Not sure about the effort, but is it possible to fill it with the dummy content we have in the integration test resources (all-config-integration-test)? https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/tree/main/cmd/monaco/test-resources/integration-all-configs/project

Then we would automatically have a working dummy setup. What do you think?

Reply:
The YAML file is preconfigured (for example):
```
config:
- auto-tag: auto-tag.json
auto-tag:
- name: auto-tag-one
```
But I think placing actually working content into the JSON is a bad idea - IMO people will undoubtedly push that config and we'll end up with misconfigured tenants.

I could add some text in their to make it clearer that they need to populate the file. For example: `Placeholder. Replace with your JSON content.`

Reply:
You are right, that would just be random configs, which would pollute to the environment of the user.

I really like the idea of the placeholder text - then its immediately clear to the user that the content of the json needs to be replaced.

Reply:
Done in `b48d8e1`